### PR TITLE
Add Eigenlayer CLI commands and integrate with blueprint manager

### LINKED ISSUE #1185: [TASK] Update eigenlayer blueprint runner to start bridge & proxy server
## Context

The eigenlayer runner is quite outdated and missed the logic to start bridge & auth server

## Task

The output of this issue is that the Eigenlayer blueprint runner successfully initiates the bridge & proxy server

Ref: [run_eigenlayer_avs](https://github.com/tangle-network/blueprint/blob/main/cli/src/command/run/eigenlayer.rs#L42)

### PR DESCRIPTION
**Pull request by drewstone**
- Eigenlayer CLI integration for registrating/deregistration/listing AVSes.
- Integrates into the blueprint manager similarly to Tangle blueprints.

## Experience
An operator can register for many AVSes through the CLI. The blueprint manager will watch a local file and check the onchain system for operator's registrations in different AVS. The blueprint manager spawns the AVSes like it does spawning blueprint instances.

## UX/Operator Experience
- Use the blueprint manager to run a single or multiple AVS, through the CLI and through the blueprint manager.
- The difference is this has NOTHING to do with Tangle Blueprints btw. This is local registration, stored locally, (and stored onchain) with a path specified for where the binary is.

## Test / logic improvements
- Error/edge cases when binary doesn't exist / after registration
- Error/edge cases/success cases when the local registration file is non-existent.
- Identify most important improvement regarding the binary. The assumption is you've built an Eigenlayer AVS with the Blueprint SDK. Now you want to share it with operators. Perhaps we add the ability to specify a github release/binary, remote URL, to fetch when registering.

## Issues
- Close: https://github.com/tangle-network/blueprint/issues/1185

### CODE REVIEW

**Review by danielbui12 — CHANGES_REQUESTED** [review]
- Many lint issues need to be fixed
- examples/incredible-squaring-eigenlayer do not work anymore, appear hang
- Eigenlayer blueprint runner crashed.
    + **SITUATION**: My machine have not install k8s, so it does not have kubctl config.
    + **ACTION**: I ran `RUST_LOG=info cargo tangle blueprint run -p eigenlayer -u http://localhost:55003/ --keystore-path ./test-keystore -b ../../target/release/incredible-squaring-blueprint-eigenlayer -d ./tmp -f settings.env`, then it crashed
        Details:
        ```
        Error: 
        0: Kubernetes: Failed to infer configuration: failed to infer config: in-cluster: (failed to read an incluster environment variable: environment variable not found), kubeconfig: (failed to read kubeconfig : No such file or directory (os error 2))
        1: Failed to infer configuration: failed to infer config: in-cluster: (failed to read an incluster environment variable: environment variable not found), kubeconfig: (failed to read kubeconfig: No such file or directory (os error 2))
        2: failed to infer config: in-cluster: (failed to read an incluster environment variable: environment variable not found), kubeconfig: (failed to read kubeconfig : No such file or directory (os error 2))
        3: failed to read kubeconfig : No such file or directory (os error 2)
        4: No such file or directory (os error 2)
        ```
    + **REASON**: Due to [`blueprint-manager`](https://github.com/tangle-network/blueprint/blob/79cfdde95be77e9acce2dd8b962a1099c6a3cbea/cli/Cargo.toml#L37) is always installed with feature container, even I use native binary
    + **PROPOSE CHANGES**:
      
          By updating [`Cargo.toml`](https://github.com/tangle-network/blueprint/blob/79cfdde95be77e9acce2dd8b962a1099c6a3cbea/cli/Cargo.toml), users can choose which features to enable on `cargo-tangle`, which selectively controls features passed down to `blueprint-manager` when install `cargo-tangle` by `cargo install cargo-tangle --features blueprint-manager-containers,blueprint-manager-tee`
      
          ```
          [package]
          name = "cargo-tangle"
          version = "0.4.0-alpha.23"
          ...
      
          [lints]
          workspace = true
      
          [dependencies]
          ...
      
          # PROPOSE CHANGE: Remove all features
          blueprint-manager = { workspace = true, default-features = false }
          ...
      
          [features]
          default = []
          vm-debug = ["blueprint-manager/vm-sandbox"]
      
      
          # PROPOSE CHANGE: The parent crate features `blueprint-manager-containers` and `blueprint-manager-tee` explicitly enable the corresponding sub-crate features.
          # PROPOSE CHANGE: Add configurable features for blueprint-manager features
          blueprint-manager-containers = ["blueprint-manager/containers"]
          blueprint-manager-tee = ["blueprint-manager/tee"]
      
          # Optional feature to enable blueprint-manager without any extra features
          blueprint-manager = ["blueprint-manager"]
      
          # PROPOSE CHANGE: Example combined feature if you want
          all-blueprint-manager = ["blueprint-manager-containers", "blueprint-manager-tee"]
          ```

File: crates/manager/tests/runtime_target_test.rs
Comment (danielbui12) [inline crates/manager/tests/runtime_target_test.rs]: TODO: Use a common function to deploy AVS before each of these test

File: crates/testing-utils/eigenlayer/src/env.rs
Comment (danielbui12) [inline crates/testing-utils/eigenlayer/src/env.rs]: I reverted the code in this file to ensure the test for `examples/incredible-squaring-eigenlayer` works correctly. The `examples/incredible-squaring-eigenlayer` already contains the deployment contract logic, so redeploying here would result in duplication. However, this change causes the new tests in `crates/manager/tests/*` to fail.

### Why should deployment not be kept here?
- It introduces significant changes to set the contract address as an environment variable.
- Each AVS contains its own specific logic, so deployment logic for a specific AVS should not be included in a common function.

**Review by danielbui12 — CHANGES_REQUESTED** [review]
- Need to create a common function to deploy a specific AVS  for tests in `crates/manager/tests`
- Need to verify CLI `cargo tangle blueprint run --eigenlayer`, `cargo tangle blueprint eigenlayer`

File: crates/testing-utils/eigenlayer/src/env.rs [hunk 1]
```diff
@@ -1,22 +1,40 @@
...
-};
-use eigenlayer_contract_deployer::deploy::{deploy_avs_contracts, DeployedContracts};
-use eigenlayer_contract_deployer::permissions::setup_avs_permissions;
+use eigensdk::utils::slashing::middleware::registry_coordinator::ISlashingRegistryCoordinatorTypes::OperatorSetParam;
+use eigensdk::utils::slashing::middleware::registry_coordinator::IStakeRegistryTypes::StrategyParams;
+use eigensdk::utils::slashing::middleware::registry_coordinator::RegistryCoordinator;
 use url::Url;
 
+/// The default Allocation Manager address
+pub const ALLOCATION_MANAGER_ADDR: Address = address!("8a791620dd6260079bf849dc5567adc3f2fdc318");
```
Comment (drewstone) [inline crates/testing-utils/eigenlayer/src/env.rs]: Are these values hardcoded anywhere else? In json or other config files? We should ensure we unify if so

File: crates/testing-utils/eigenlayer/src/env.rs
Comment (drewstone) [reply crates/testing-utils/eigenlayer/src/env.rs]: Please review the eigensdk-rs repo more too, a lot of this was taken from how they test their SDK. Changes like this should be done as thoughtfully as possible.

File: crates/testing-utils/eigenlayer/src/env.rs (hunk 1 above)
Comment (danielbui12) [reply crates/testing-utils/eigenlayer/src/env.rs]: @drewstone these are from [dumped state](https://github.com/tangle-network/blueprint/blob/b4901c95f3f2d125b1df5643d0981e68862f8707/crates/chain-setup/anvil/data/state.json), containing eigenlayer smart contract and similar to  values at [here](https://github.com/tangle-network/blueprint/blob/f4e68b9238d035d6b5791cf7c60dc05b064eedbd/crates/runner/src/eigenlayer/config.rs#L112-L136). btw I'll fix this

**Review by danielbui12 — APPROVED** [review]

**Comment by drewstone** [comment]
@claude review
